### ISSUE TITLE
Commercial Usage of Pre-Trained Models

### ISSUE THREAD

**Issue by nepes-nepesov** [KaiyangZhou/deep-person-reid#520]
Is commercial usage of pre-trained models from the the [model zoo](https://kaiyangzhou.github.io/deep-person-reid/MODEL_ZOO) allowed? What is their license?

**Comment by KaiyangZhou** [comment]
See https://github.com/KaiyangZhou/deep-person-reid/blob/master/LICENSE

**Comment by nepes-nepesov** [comment]
Thank you for the link, I've already seen that license. Doesn't this license cover just the code and not the pre-trained models? Don't we have to take into account the license of the data used for training these models?

**Comment by KaiyangZhou** [comment]
Yes, you're right. Some data might not be available for commercial use, so the pre-trained models are supposed to be used for research purpose only.
